composer.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "ubmannheim/publist4ubma2",
+    "type": "typo3-cms-extension",
+    "description": "TYPO3 extension which shows publications from an EPrints repository",
+    "keywords": [
+        "TYPO3",
+        "extension",
+        "EPrints"
+    ],
+    "homepage": "https://github.com/UB-Mannheim/publist4ubma2",
+    "support": {
+        "issues": "https://github.com/UB-Mannheim/publist4ubma2/issues",
+        "wiki": "https://github.com/UB-Mannheim/publist4ubma2/wiki"
+    },
+    "authors": [
+        {
+            "name": "Sebastian Kotthoff",
+            "email": "sebastian.kotthoff@rz.uni-mannheim.de",
+            "role": "Developer",
+            "homepage": "http://dws.informatik.uni-mannheim.de/en/people/administration/sebastiankotthoff/"
```

Comment:
The `http` version as given in the composer.json is working for me. Or was this a general complain that the website should support `https` also?